sdk/provisioning/Azure.Provisioning/src/Expressions/BicepTypeMapping.cs

```diff
@@ -96,6 +98,11 @@ public static BicepExpression ToBicep(object? value, string? format)
             // in bicep source and you need to use a parameter file for larger
             // values
             long i => BicepSyntax.Value((int)i),
```

Comment:
FWIW, I wonder if we'll have the same bug here in the future?  I haven't seen anyone trying to use values larger than `int.MaxValue` by parsing a JSON string though.  I don't think we should fix it now, but keep it in mind if we ever hear any customer issues.

Reply:
yep, I wonder the same thing for long. I will keep it in mind